.travis.yml

```diff
@@ -25,6 +25,8 @@ matrix:
 before_script:
   - bash tests/bin/install.sh wordpress_test root '' localhost $WP_VERSION
   - bash tests/bin/travis.sh before
-script: phpunit -c phpunit.xml
+script:
+  - phpunit -c phpunit.xml
+  - vendor/bin/phpstan analyze
```

Comment:
I have been using phpstan during local development, and [Scrutinizer-CI](https://scrutinizer-ci.com/g/pryley/site-reviews/) for CI. I'm not sure what the benefit would be to run phpstan in CI also...

Reply:
Okay, you may remove it.